community/contributors.md

```diff
@@ -23,17 +23,15 @@ There is one other way to contribute, and it's the most important: **use the wor
 
 Here is the link to the Baseline Protocol code of conduct:
 
-{% embed url="https://github.com/ethereum-oasis/baseline/blob/master/CODE_OF_CONDUCT.md" %}
+{% embed url="https://github.com/ethereum-oasis-op/baseline/blob/master/CODE_OF_CONDUCT.md" %}
 
 ## Technical Contributors <a href="#technical-contributors" id="technical-contributors"></a>
 
 Technical contributors either are working on architecture or developing code...but even correcting the language of documentation counts as a technical contribution and qualifies you to vote in upcoming [TSC](community-leaders.md#your-technical-steering-committee) elections.
 
 ## Core Developers <a href="#maintainers" id="maintainers"></a>
 
-Once you've done some work as a Member, you may wish to become a **Core developer** and have a direct hand in deciding what work is merged to the Main/Master Branch to become official Baseline Protocol technology and specifications.
-
-Here's a list of [current Core Developers](broken-reference).
+Once you've done some work as a Member, you may wish to become a **Maintainer** and have a direct hand in deciding what work is merged to the Main/Master Branch to become official Baseline technology and specifications.
```

Comment:
Given that something like content is in MAINTAINERS.md, why not leave that as a "single source of truth", and either link to it or embed it here?

Reply:
Done